src/pages/api/disable-draft.page.tsx

```diff
@@ -1,7 +1,25 @@
+import { COOKIE_NAME_PRERENDER_BYPASS } from 'next/dist/server/api-utils';
+
 export default async function exit(_, res) {
-  // Exit the current user from "Draft Mode". This function accepts no args.
+  // Exit the current user from "Draft Mode".
   res.setDraftMode({ enable: false });
 
+  // Override cookie header for draft mode for usage in live-preview
+  // https://github.com/vercel/next.js/issues/49927
+  // https://github.com/vercel/next.js/blob/62af2007ce78fdbff33013a8145efbcacbf6b8e2/packages/next/src/server/api-utils/node.ts#L293
+  const headers = res.getHeader('Set-Cookie');
+  if (Array.isArray(headers)) {
```

Comment:
If you are so inclined you could move this out to a util function, as it's reused

Reply:
thought about this too, but hope we can remove it shortly